Assets/Shopify/UIToolkit/Interfaces/IShopCredentials.cs

```diff
@@ -1,7 +1,8 @@
 ﻿namespace Shopify.UIToolkit {
     public interface IShopCredentials {
         ShopCredentialsVerificationState CredentialsVerificationState { get; set; }
-        string ShopDomain { get; }
-        string AccessToken { get; }
+
+        string GetShopDomain();
```

Comment:
I didn't really want to do this but when I defined the properties to match the interface like below:

```csharp
public string AccessToken; // Unity likes this
public string AccessToken { get; set; } // No bueno
```

The `PropertyField` on the serializeObject didn't like that because I guess Unity treats the second case not as a field.

Reply:
yea this is true... Good solution!